war/src/main/webapp/WEB-INF/elasticsearch.yml

```diff
@@ -31,39 +33,20 @@ path.home: es
 #
 path.data: ${exo.data.dir}
 
-# Path to where plugins are installed:
-#
-path.plugins: es/plugins
```

Comment:
Yes, this property/capability was removed:
https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_50_plugins.html